launch/world_modeling/world_model.launch

```diff
@@ -1,47 +1,23 @@
-	<arg name="robot_real" value="$(optenv ROBOT_REAL false)" />
-	<arg name="faces_db" default="" />
-
-	<include file="$(env ROBOT_BRINGUP_PATH)/machines/$(arg machine).machine" />
-
-	<arg name="config" default="world_modeling/world_model.yaml"/>
-
-	<!-- World model -->
-	<node name="ed" pkg="ed" type="ed" output="log" machine="$(arg machine)" />
-
-	<!-- Configure world model -->
-	<node name="ed_configure" pkg="ed" type="configure" args="$(env ROBOT_BRINGUP_PATH)/parameters/$(arg config)" output="log" machine="$(arg machine)"/>
-
-	<!-- Face recognition -->
-	<group ns="face_recognition">
-		<node name="face_recognition" pkg="image_recognition_openface" type="face_recognition_node" output="log" machine="$(arg machine)">
-			<rosparam command="load" file="$(env ROBOT_BRINGUP_PATH)/parameters/world_modeling/face_recognition.yaml" />
-			<param name="topic_save_images" value="false" />
-			<param name="service_save_images" value="$(arg robot_real)" />
-			<param name="save_images_folder" value="~/MEGA/data/$(arg robot_env)/training_data" />
-			<param name="db" value="$(arg faces_db)" />
-		</node>
-
-		<node pkg="image_recognition_keras" type="face_properties_node" name="keras_face_properties">
-			<param name="weights_file_path" value="~/data/keras_models/age_gender/weights.28-3.73.hdf5" />
-			<param name="save_images" value="$(arg robot_real)" />
-			<param name="save_images_folder" value="~/MEGA/data/$(arg robot_env)/training_data" />
-		</node>
-	</group>
-
-    <!-- Object recognition -->
-    <group ns="object_recognition">
-        <node name="object_recognition" pkg="image_recognition_tensorflow" type="object_recognition_node" output="log" machine="$(arg machine)">
-            <param name="graph_path" value="~/MEGA/data/$(arg robot_env)/models/image_recognition_tensorflow/output_graph.pb" />
-            <param name="labels_path" value="~/MEGA/data/$(arg robot_env)/models/image_recognition_tensorflow/output_labels.txt" />
-            <param name="save_images" value="$(arg robot_real)" />
-            <param name="save_images_folder" value="~/MEGA/data/$(arg robot_env)/training_data" />
-        </node>
-    </group>
+    <!-- Get the machine file -->
+    <arg name="machine" default="localhost"/>
+    <include file="$(env ROBOT_BRINGUP_PATH)/machines/$(arg machine).machine" />
+
+    <arg name="robot_real" value="$(optenv ROBOT_REAL false)" />
+
+    <arg name="robot_env" default="$(optenv ROBOT_ENV robotics_testlabs)" />
+
+    <arg name="config" default="world_modeling/world_model.yaml"/>
+    <arg name="config_sim" default="$(arg config)"/>
+
+    <!-- World model -->
+    <node name="ed" pkg="ed" type="ed" output="log" machine="$(arg machine)" />
+
+    <!-- Configure world model -->
+    <node name="ed_configure" pkg="ed" type="configure" args="$(env ROBOT_BRINGUP_PATH)/parameters/$(arg config)" output="log" machine="$(arg machine)" if="$(optenv ROBOT_REAL false)" />
```

Comment:
Why configure both normal as well as sim params. Doesn't seem like the way to go

Reply:
Removed the config_sim